aii-ks/src/main/perl/ks.pm

```diff
@@ -692,8 +692,14 @@ sub log_action {
     }
 
     if ($consolelogging) {
+        # In EL7, /dev/console doesn't add the carriage return for each line feed (new line)
+        # resulting in unreadable messages on the console. /dev/pts/0 must be used instead.
+        # But on prior versions, /dev/pts/0 doesn't exist at installation time and cannot be used.
+        # The following code allows to use /dev/pts/0 if it exists else to revert to /dev/console.
+        push(@logactions, "console='/dev/console'");
```

Comment:
I am not sure to understand your point. Why it would be clearer to have it earlier? This variable definition in the %pre script is useful only if you use console logging, there is  no real point in adding them otherwise. I would find it more confusing to have this defined unconditionally, both in the ks.pm code and the resulting %pre script...